### PR TITLE
[QoS]Test_buffer_model test skip for Nokia platform

### PR DESCRIPTION
**Pull request by ansrajpu-git**
### Description of PR

Updating skip condition for test_buffer_model_test.py in conditional mark as this was overriding the base test skip condition for  nokia platform 'x86_64-nokia_ixr7250e_36x400g-r0'

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
Test suite was executing skipped test.
#### How did you do it?
Updated the conditional mark skip condition
#### How did you verify/test it?
Executed the test and verify the results

### CODE REVIEW

**Review by wsycqyz — APPROVED** [review]
Look good to më

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -1166,7 +1166,7 @@ qos/test_buffer.py::test_buffer_model_test:
   skip:
     reason: "Running only on mellanox devices and covered by unit testing"
     conditions:
-      - "asic_type in ['mellanox']"
+      - "asic_type in ['mellanox'] or platform in ['x86_64-nokia_ixr7250e_36x400g-r0']"
```
Comment (arlakshm) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1169]: is this an asic limitation? can we add asic_subtype == 'dnx' iso of platform check?

**Review by arlakshm — APPROVED** [review]

**Comment by ansrajpu-git** [comment]
@arlakshm , updated condition. please approve.

**Review by vmittal-msft — APPROVED** [review]

**Comment by vmittal-msft** [comment]
@yxieca @wangxin please help merge

**Review by arlakshm — APPROVED** [review]

**Pull request merged**